src/app/loadout-builder/ClosableContainer.tsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import styles from './ClosableContainer.m.scss';
+import clsx from 'clsx';
 
 /**
  * A generic wrapper that adds a "close" button in the top right corner.
  */
 export default function ClosableContainer({
   children,
   enabled = true,
+  showCloseIconOnHover = false,
   onClose,
 }: {
   children: React.ReactNode;
   enabled?: boolean;
+  showCloseIconOnHover?: boolean;
```

Comment:
This will never be undefined because it has a default value

Reply:
That is true but the ts compiler doesn't let you use the component without passing it in as a prop. This lets you ignore the prop. It's why enabled is also optional.

<img width="1064" alt="Screen Shot 2020-08-25 at 8 05 30 pm" src="https://user-images.githubusercontent.com/7344652/91161632-5895eb80-e70e-11ea-967a-5dc8be2eb242.png">